src/vm/codeversion.h

```diff
@@ -105,14 +105,16 @@ class NativeCodeVersion
         Synthetic
     };
 
+    struct SyntheticStorage
+    {
+        PTR_MethodDesc m_pMethodDesc;
+    };
+
```

Comment:
I also fixed this.

Reply:
I used this approach because we made similar fixes last year for nested types: https://github.com/dotnet/coreclr/pull/18591.

Reply:
@noahfalk what would you prefer here - extracting the struct SyntheticStorage as a standalone type or removing the SyntheticStorage type name from the struct member in the union?

Reply:
Syncd with master and it went away. @janvorli, can you please take another look at the changes?